app/templates/components/filter-by.hbs

```diff
@@ -1,10 +1,61 @@
-{{#if isShowingBody}}
-  <div class="filter-table">
-    <ul>
-      <li>Teams</li>
+{{!-- Filter By --}}
+
+<div class="filter">
+  <BsDropdown as |dd|>
+    <dd.toggle class="filter-by"><span>Filter by</span><p
+        class="emp"
+        {{action "filterby"}}
+      >{{if currentTeam currentTeam "All Employees"}}</p>
+      <img
+        src="/assets/images/accordion-open.svg"
+        alt="img"
+        class="open {{if clicked 'turn'}}"
+      />
+    </dd.toggle>
+    <dd.menu>
+      <li class="team">Teams</li>
       {{#each myTeamMembers as |myTeam|}}
-        <li {{action "isFilter" myTeam.team}}>{{myTeam.team}}</li>
+        <li
+          class="filter-table"
+          {{action "isFilter" myTeam.team}}
+        >{{myTeam.team}}</li>
       {{/each}}
-    </ul>
-  </div>
-{{/if}}
\ No newline at end of file
+    </dd.menu>
+  </BsDropdown>
+</div>
+
+{{!-- Sort By --}}
+
+<div class="sort">
+  <BsDropdown as |dd|>
+    <dd.toggle class="filter-by"><span>Sort by</span><p
+        class="emp"
+        {{action "sortby"}}
+      >{{if sortDisplay sortDisplay "First Name"}}</p>
+      <img
+        src="/assets/images/accordion-open.svg"
+        alt="img"
+        class="open {{if sortByclicked 'turn'}}"
```

Comment:
Instead of having one more image and class to show the up and down arrow, You can use the bs-dropdown arrow itself right..? You can change the styles based on the mock.

Reply:
waiting for your solution